src/ocm/components/hosts/use-hosts-table.tsx

```diff
@@ -180,6 +180,34 @@ export const useHostsTable = (cluster: Cluster) => {
       [cluster.id, dispatch],
     );
 
+  const updateDiskSkipFormatting = React.useCallback(
+    async (doFormatDisk: boolean, hostId: Host['id'], diskId: Disk['id']) => {
+      try {
+        if (!cluster) {
+          return new Error(`Cannot update disksSkipFormatting in host ${hostId}\nMissing cluster`);
+        }
+        if (!diskId) {
+          throw new Error(`Cannot update disksSkipFormatting in host ${hostId}\nMissing diskId`);
+        }
+        const { data } = await HostsService.updateFormattingDisks(
+          cluster.id,
+          hostId,
+          diskId,
+          !doFormatDisk,
+        );
+        resetCluster ? void resetCluster() : dispatch(updateHost(data));
+      } catch (e) {
+        handleApiError(e, () =>
+          addAlert({
+            title: 'Failed to set disks skip formatting',
+            message: getApiErrorMessage(e),
+          }),
+        );
+      }
+    },
+    [dispatch, resetCluster, addAlert, cluster],
```

Comment:
We don't need the entire `cluster` object, please make it depend only on `cluster.id` and use only `cluster.id` inside the function